tools/tblastn.md

```diff
@@ -0,0 +1,72 @@
+# {TBLASTN}
+written by: [ZhenghaoXiao](https://github.com/ZhenghaoXiao)
+
+[{15} minutes] {`Translated BLAST(TBLASTN)` is a website based tool on NCBI that can compare `protein sequences` against `nucleotide sequences` to identify `translated nucleotide database sequences` similar to the provided `protein query`, which
+ allows for functional and evolutionary studies of viruses.}
+
+**Tutorial Objective**: This tutorial will teach how to look for translated nucleotide database sequences similar to the protein query.
+
+## Input / Prerequisites
+- [Tool Weblink](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=tblastn&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome)
+- [Link to example data](https://www.ncbi.nlm.nih.gov/protein/KAI4295448.1)--`Bauhinia variegata`--`I did not use my obelisk 17 sequence because the translated protein sequence did not provide me any result on tblastn`
+- A FASTA Protein Query Sequence
+- Adjustment on the Parameters-E value(Expect threshold), Max Target Sequences, Database Choice, Organisms, etc
+
+## Output
+
+A `table` showing matches between the `query protein` and list of `translated nucleotide sequences` that produce the most significant alignment, ranking from the highest E-value to the lowest. The graphic summary, detailed alignment, and taxonomy will also be provided.
+
+### 1. 
+Go to the TBLASTN website(https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=tblastn&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome)
+
+### 2. 
+Then go to the example data website, copy and paste the protein query to the NCBI TBLASTN website input `Enter Query Sequence`.
+
+![](img/tblastn/1.png)
```

Comment:
Best practice is to provide "Figure Titles" so in the case the image is not loading then text (Figure Title) is at least displayed.